src/clojure_by_example/ex05_immutability_and_fp.clj

```diff
@@ -118,7 +114,7 @@ habitable-planets ; confirm by checking the value of this
 ;; At each re-definition, `am-i-mutable` effectively "becomes"
 ;; the new _immutable_ value.
 ;;
-;; "Immutable" value is actually an oxymoron. Recall, we said that
```

Comment:
`"Immutable" value` is a tautology. `"Mutable" value` is an oxymoron. We can choose to use one or the other.

Reply:
Ayyo! Yes, /s/Immutable/Mutable